traffic_ops/traffic_ops_golang/crconfig/deliveryservice.go

```diff
@@ -32,16 +32,29 @@ import (
 	"github.com/lib/pq"
 )
 
-const CDNSOAMinimum = 30 * time.Second
-const CDNSOAExpire = 604800 * time.Second
-const CDNSOARetry = 7200 * time.Second
-const CDNSOARefresh = 28800 * time.Second
-const CDNSOAAdmin = "traffic_ops"
+// Start-Of-Authority record fields; see RFC1035 section 3.3.13.
+// (https://tools.ietf.org/html/rfc1035#section-3.3.13).
+const (
+	CDNSOAAdmin   = "traffic_ops"
+	CDNSOAExpire  = 604800 * time.Second
```

Comment:
It can, but I assume it isn't because the actual Expire field of an SOA record is given in seconds so this gives you a better idea of what is literally inserted.

Reply:
Good point, seconds makes sense here. It can still be factored, since it's not immediately apparent that `604800` is 1 week in seconds.